src/Commands/RegenerateCommand.php

```diff
@@ -42,21 +42,27 @@ public function handle()
             return;
         }
 
-        $this->getMediaToBeRegenerated()->each(function (Media $media) {
+        $mediaFiles = $this->getMediaToBeRegenerated();
+
+        $bar = $this->output->createProgressBar($mediaFiles->count());
+
+        $mediaFiles->each(function (Media $media) use ($bar) {
             try {
                 $this->fileManipulator->createDerivedFiles($media);
                 $this->info("Media {$media->id} regenerated");
```

Comment:
This will need to be removed otherwise it'll mess up the output of the progress bar unless I'm mistaken (since you're technically outputting things in the middle of outputting the bar)